helpers/middleman/space.rb

```diff
@@ -1,38 +1,36 @@
-load File::expand_path('./page.rb', __dir__)
+load File.expand_path('./page.rb', __dir__)
 
 module MiddlemanSpaceHelpers
   include MiddlemanPageHelpers # for locale_page method
 
   def is_category_page?
     category = current_page.metadata[:locals][:category]
     return true if category && category[:id] && category[:id].downcase != 'all'
-    return false
+
+    false
   end
 
   def space_category_title
     # NOTE: This could fail on a partner called 'blog' or a blog titled 'Partners are the the way of the future, man'
     # That's why blog comes first, I guess.
     # Ideally, I would ignore localization paths and then expect the first one to be blog/partners..
-    space = if current_path_array.any?{ |path| /^blog/ =~ path }
-      'Blog'
-    elsif current_path_array.any?{ |path| /^partners/ =~ path }
-      'Partners'
-    end
-
+    space = 'Partners'
```

Comment:
I'd just keep the logic before, just incase…maybe note that blogs are gone, but it's unclear.

```suggestion
    space = if current_path_array.any?{ |path| /^partners/ =~ path }
      'Partners'
    end
```